Extend/Language.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ *
+ */
+
+namespace Wirecard\Oxid\Extend;
+
+/**
+ * Extends the OXID Language class
+ *
+ * @mixin \OxidEsales\Eshop\Core\Language
+ *
+ * @since 1.0.1
+ */
+class Language extends Language_parent
+{
+    const TRANSLATION_KEY_PREFIX = 'wdpg_';
+    const FALLBACK_LANGUAGE_ABBR = 'en';
```

Comment:
I would rather call it `FALLBACK_LANGUAGE_CODE` in order to avoid cut-off variable names.

Reply:
It fits the compared value and oxid implementation